app/javascript/controllers/map_controller.js

```diff
@@ -1,64 +1,35 @@
-import { Controller } from "@hotwired/stimulus"
-import mapboxgl from 'mapbox-gl'
+import { Controller } from "@hotwired/stimulus";
+import mapboxgl from "mapbox-gl";
 
 // Connects to data-controller="map"
 export default class extends Controller {
   static values = {
     apiKey: String,
     markers: Array,
-    marker: Object
-  }
+  };
 
   connect() {
-    mapboxgl.accessToken = this.apiKeyValue
-
-    //Singular Location for the show page
+    mapboxgl.accessToken = this.apiKeyValue;
     this.map = new mapboxgl.Map({
       container: this.element,
       style: "mapbox://styles/mapbox/streets-v10",
-      center: [this.markerValue.lng, this.markerValue.lat],
-      zoom: 14
-    })
-
-    //Multiple locations for the home page
-    this.map = new mapboxgl.Map({
-      container: this.element,
-      style: "mapbox://styles/mapbox/streets-v10"
-    })
-    //Multiple
-    this.#addMarkersToMap()
-    this.#fitMapToMarkers()
-
-    //Singular
-    this.#addMarkerToMap()
-    this.#fitMapToMarker()
+    });
+    this.#addMarkersToMap();
```

Comment:
I refactored this page, removing duplicate methods, and ensuring the data used is an array, instead of a string. this keep the page simple and easier to work with.